.git-blame-ignore-revs

```diff
@@ -1,7 +1,19 @@
 # enforce unified code style with black
+# https://github.com/pyvista/pyvista/pull/2180
 28914fc2236cd5850f536b151d0ef383855d8519
+# https://github.com/pyvista/pyvista/pull/2212
 db0691cfb6b033e55664ffba25deb456a386a48c
 
 # enforce unified code style in our documentation with blackdoc
+# https://github.com/pyvista/pyvista/pull/4119
 a26a52d037a265774be0ca06bb141e3346e73ad0
+# https://github.com/pyvista/pyvista/pull/4129
 635052d1bce8615fcc4879f5f200921a4f78aa1a
+
+# ruff formatting
+# https://github.com/pyvista/pyvista/pull/6607
+7c4166477b875d40e4650b102f67a29a40c2d8f2
+# https://github.com/pyvista/pyvista/pull/6632
+7f8ba86bbe07f85380937bc8d7561775e6a1d4d8
```

Comment:
I learned from https://github.com/pyvista/pyvista/pull/5059#issuecomment-1760334221 that the criterion for adding to .git-blame-ignore-revs is whether or not it is automatically formatted, rather than the number of lines. For that reason, I think it would be better to delete this commit as it is human handed change.
```suggestion
```